wiki/osu!_rankings_history/en.md

```diff
@@ -0,0 +1,330 @@
+
+## 2009
+
+| Player | From | To | Sources |
+| --: | :-- | :-- | :-- |
+| ![][flag_US] [Rolled](https://osu.ppy.sh/users/5243) | *2008-12-11* | 2009-01-19 |  |
+| ![][flag_US] [Cyclone](https://osu.ppy.sh/users/18589) | 2009-01-19 | 2009-03-05  |  |
+| ![][flag_PL] [Kubu](https://osu.ppy.sh/users/29130) | 2009-03-05  | 2009-03-06  |  |
+| ![][flag_US] [Cyclone](https://osu.ppy.sh/users/18589) | 2009-03-06 | March 2009  |  |
+| ![][flag_JP] [tobebuta](https://osu.ppy.sh/users/14053) | March 2009 | 2009-03-18 |  |
+| ![][flag_US] [Cyclone](https://osu.ppy.sh/users/18589) | 2009-03-18 | 2009-04-16 |  |
+| ![][flag_PL] [Kubu](https://osu.ppy.sh/users/29130) | 2009-04-16 | 2009-04-18 |  |
+| ![][flag_US] [Cyclone](https://osu.ppy.sh/users/18589) | 2009-04-18 | June 2009 |  |
+| ![][flag_JP] [tobebuta](https://osu.ppy.sh/users/14053) | June 2009 | July 2009 |  |
+| ![][flag_US] [Cyclone](https://osu.ppy.sh/users/18589) | July 2009 | 2009-07-05 |  |
+| ![][flag_GB] [Doomsday](https://osu.ppy.sh/users/18983) | 2009-07-05 | July 2009 |  |
+| ![][flag_UY] [H1ko](https://osu.ppy.sh/users/58710) | July 2009 | 2009-08-13 |  |
+| ![][flag_PL] [WubWoofWolf](https://osu.ppy.sh/users/39828) | 2009-08-13 | 2009-09-03 |  |
+| ![][flag_UY] [H1ko](https://osu.ppy.sh/users/58710) | 2009-09-03 | October 2009 |  |
+| ![][flag_PL] [WubWoofWolf](https://osu.ppy.sh/users/39828) | October 2009 | November 2009 |  |
+| ![][flag_GB] [Doomsday](https://osu.ppy.sh/users/18983) | November 2009 | *2010-02-07* |  |
+
+## 2010
+
+| Player | From | To | Sources |
+| --: | :-- | :-- | :-- |
+| ![][flag_GB] [Doomsday](https://osu.ppy.sh/users/18983) | *November 2009* | 2010-02-07 |  |
+| ![][flag_UY] [H1ko](https://osu.ppy.sh/users/58710) | 2010-02-07 | 2010-07-10 |  |
+| ![][flag_KR] [Sure](https://osu.ppy.sh/users/70730) | 2010-07-10 | 2010-08-10 |  |
+| ![][flag_JP] [zaru-d](https://osu.ppy.sh/users/79678) | 2010-08-10 | September 2010 |  |
+| ![][flag_PL] [Kubu](https://osu.ppy.sh/users/29130) | September 2010 | October 2010 |  |
+| ![][flag_GB] [Doomsday](https://osu.ppy.sh/users/18983) | October 2010 | November 2010 |  |
+| ![][flag_JP] [zaru-d](https://osu.ppy.sh/users/79678) | November 2010 | November 2010 |  |
+| ![][flag_JP] [Flute](https://osu.ppy.sh/users/211278) | November 2010 | 2010-12-08 |  |
+| ![][flag_GB] [Doomsday](https://osu.ppy.sh/users/18983) | 2010-12-08  | *2011-01-09* |  |
+
+## 2011
+
+| Player | From | To | Sources |
+| --: | :-- | :-- | :-- |
+| ![][flag_GB] [Doomsday](https://osu.ppy.sh/users/18983) | *2010-12-08* | 2011-01-09 |  |
+| ![][flag_CN] [Ana_Coppola](https://osu.ppy.sh/users/88717) | 2011-01-09 | January 2011 |  |
+| ![][flag_GB] [Doomsday](https://osu.ppy.sh/users/18983) | January 2011 | January 2011 |  |
+| ![][flag_JP] [Flute](https://osu.ppy.sh/users/211278) | January 2011 | 2011-02-01 |  |
+| ![][flag_GB] [Doomsday](https://osu.ppy.sh/users/18983) | 2011-02-01 | 2011-03-20 |  |
```

Comment:
```suggestion
| ![][flag_JP] [Flute](https://osu.ppy.sh/users/211278) | November 2010 | 2010-12-08 |  |
| ![][flag_GB] [Doomsday](https://osu.ppy.sh/users/18983) | 2010-12-08  | *2011-01-09* |  |

## 2011

| Player | From | To | Sources |
| --: | :-- | :-- | :-- |
| ![][flag_GB] [Doomsday](https://osu.ppy.sh/users/18983) | *2010-12-08* | 2011-01-09 |  |
| ![][flag_CN] [Ana_Coppola](https://osu.ppy.sh/users/88717) | 2011-01-09 | January 2011 |  |
| ![][flag_GB] [Doomsday](https://osu.ppy.sh/users/18983) | January 2011 | January 2011 |  |
| ![][flag_JP] [Flute](https://osu.ppy.sh/users/211278) | January 2011 | 2011-02-01 |  |
| ![][flag_GB] [Doomsday](https://osu.ppy.sh/users/18983) | 2011-02-01 | 2011-03-20 |  |
```
```suggestion
| ![][flag_JP] [Flute](https://osu.ppy.sh/users/211278) | November 2010 | 2010-11-22 |  |
| ![][flag_GB] [Doomsday](https://osu.ppy.sh/users/18983) | 2010-11-22  | *2011-01-09* |  |

## 2011

| Player | From | To | Sources |
| --: | :-- | :-- | :-- |
| ![][flag_GB] [Doomsday](https://osu.ppy.sh/users/18983) | *2010-11-22* | 2011-03-20 |  |
```

With no evidence to Ana_Coppola's or Flute's reigns here, and overwhelming evidence Doomsday held #1 throughout this period, these 2 should be removed until there's some sort of proof that these reigns actually existed.

Sources:
https://imgur.com/VlP4LOb
https://imgur.com/BTGUay5
https://imgur.com/isQQGTh
10 Years of osu!: A Retrospective: https://www.youtube.com/watch?v=0T5F0Zzy6ps (relevant screenshot at 13:05)
https://imgur.com/ZTMfOai
https://imgur.com/CIoU29J